Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -13,7 +13,7 @@
 
 # disable insecure warnings
 requests.packages.urllib3.disable_warnings()
-resilient.co3.LOG.disable(logging.ERROR)
+# resilient.co3.LOG.disable(logging.ERROR)
```

Comment:
Is it not necessary anymore?

Reply:
This line was added recently and is causing an error. I will handle it soon